thoth-client/src/queries.rs

```diff
@@ -167,3 +171,12 @@ impl From<work_query::LanguageRelation> for LanguageRelation {
         }
     }
 }
+
+// Convert generated GraphQL LocaleCode into API LocaleCode
+impl From<work_query::LocaleCode> for LocaleCode {
+    fn from(value: work_query::LocaleCode) -> Self {
+        // work_query::LocaleCode Debug prints the enum variant (e.g. "EN_US")
+        // LocaleCode derives EnumString with UPPERCASE, so this parses reliably
+        LocaleCode::from_str(&format!("{value:?}")).unwrap_or_default()
```

Comment:
Use `Display` rather than `Debug` (i.e. `to_string`).

Don't use `unwrap_or_default`, this should be a 1:1 mapping. Use `unreachable!` for the `work_query::LocaleCode::Other` variant